tests/testthat/test-lines.R

```diff
@@ -16,3 +17,23 @@ test_that("lines don't have fill", {
   })
   expect_equal(xml_attr(xml_find_one(x, ".//polyline"), "fill"), "none")
 })
+
+
+test_that("lines lty becomes stroke-dasharray", {
+  # base graphics
+  x <- xmlSVG(plot(1:3, lty = 1, type = "l"))
+  expect_equal(
+    xml_attr(suppressWarnings(xml_find_one(x,"//polyline")), "stroke-dasharray"),
+    NA_character_
+  )
+
+  dash_array <- function(lty) {
+    x <- xmlSVG(plot(1:3, lty = lty, type = "l"))
+    xml_attr(suppressWarnings(xml_find_one(x, "//polyline")), "stroke-dasharray")
```

Comment:
more than one polyline so `xml_find_one` returns warning.  I'll change to use miniplot.